### DISCUSSION TITLE
When calculating the skill damage, if the skill has a DEF debuff is it added to the calculation?

### DISCUSSION THREAD

**Discussion by shimworks** [lonqie/SchaleDB#57] [Questions] [answered]
Kasumi EX has a DEF Debuff, I'm not sure if the def Debuff is added on the first hit or if it is after.
If it is first, Is the damage calculated with the debuff or I need to add it on the debuff tab?
![image](https://github.com/lonqie/SchaleDB/assets/51913616/05898ec9-33a4-4a20-84c2-f92115a1cbda)

**Comment by lonqie** [comment, marked as answer]
Calculated skill damage only includes buffs/debuffs that you explicitly set against the student and target. The stats you see in the tables are what is used for calculation. (Clicking the wand next to enemy debuffs will add the all the current student's debuff skills automatically.)

Kasumi's EX skill applies the debuff before the damage hits
